docs/aggregation.md

```diff
@@ -1,12 +1,34 @@
-# qgrid 
+# qgrid
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows='data'>
+	<q-grid:columns>
+		<q-grid:column key="Name" title="Name" aggregation="count"></q-grid:column>
+		<q-grid:column key="Age" title="Age" aggregation="avg"></q-grid:column>
+		<!-- max || min || sum || count || avg -->
+	</q-grid:columns>
+	<!-- naming: aggregation, aggregation-footer -->
+	<q-grid:aggregation></q-grid:aggregation>
```

Comment:
Actually we are going to have 2 types of agg row and cilumn. Column in the footer, row when grouping will be applied